src/lib/Transaction.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+    import { writable } from 'svelte/store';
+    import * as StellarSdk from 'stellar-sdk';
```

Comment:
We discussed this and I said I prefer each dependency to be imported individually as to not obscure the actual dependencies. Import {Transaction, xdr} etc. from 'stellar-sdk'